deploy/kiali/kiali_cr_dev.yaml

```diff
@@ -24,3 +24,5 @@ spec:
     image_version: $KIALI_IMAGE_VERSION
     namespace: $NAMESPACE
     service_type: $SERVICE_TYPE
+    logger:
+      log_level: info
```

Comment:
can you add this to the service mesh version of this file also to maintain consistency across the two dev files? (that version of the file is used when you use make to install Kiali in an OSSM environment). See https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr_dev_servicemesh.yaml

Reply:
sure, done